sdk/textanalytics/ai-text-analytics/src/generated/lro/constants.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+export const terminalStates = ["succeeded", "failed", "canceled", "cancelled"];
```

Comment:
@joheredi why there is `"canceled"` and `"cancelled"` here?

Reply:
I took this from `@azure/core-arm` and I think the reason behind having both is that, both seem to be valid spellings depending on the region, so we wanted to be able to handle the `canceled` state regardless of the spelling